test/features/support/env.rb

```diff
@@ -1,10 +1,12 @@
+require 'dotenv'
```

Comment:
Oh, of course - once 'web' is required it will have already accessed the .env file. Good catch.